metabase/metabase
# Add UI directory for design system components and intro doc

Scaffolds out the `./frontend/src/metabase/ui` directory and some initial documentation that will appear in storybook that covers some of the basics on the effort to use Mantine for our component library.

To get the `.mdx` file to show up properly in the storybook sidebar I had to name it `Intro.stories.mdx` which was a bit of a bummer, but we can solve that later. Also needed to add the `@storybook/blocks` package to be able to use the `Meta` tag for the page title and some other goodies that'll be useful subsequently.

## How to verify
- Spin up storybook locally with `yarn run storybook` and you should see an `Intro` docs page in the lefthand sidebar.

<img width="1392" alt="Screen Shot 2023-06-01 at 11 42 15 AM" src="https://github.com/metabase/metabase/assets/5248953/93a24176-1363-4ab9-979f-4fd62ac645c2">
